### ISSUE TITLE
Edit Pool Page - h2 cannot be a child of h2

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#4247]
![image](https://user-images.githubusercontent.com/8978655/194640297-98743bee-a8e9-4a29-82b3-033fd26fbca8.png)
